### PR TITLE
Threats Obstacle Avoidance

### PR DESCRIPTION
**Pull request by gtrabes**

### CODE REVIEW

File: saviSimulation/config.cfg [hunk 1]
```diff
@@ -14,10 +14,10 @@ RANDOM_SEED = 137
...
 HOUSE_SIZE = 15
-THREAT_SIZE = 10
+THREAT_SIZE = 15
 ANTENNA_SIZE = 26
 UGV_SIZE = 26
 UAV_SIZE = 26
-SENSORS_ERROR_PROB = 0.3
-SENSORS_ERROR_STD_DEV = 20
+SENSORS_ERROR_PROB = 0
+SENSORS_ERROR_STD_DEV = 0
```
Comment (pgavigan) [inline saviSimulation/config.cfg]: I suspect that we don't actually want these changes to the config file. Do we?

File: saviSimulation/savi/jason_processing/UaVBehavior.java [hunk 2]
```diff
@@ -36,7 +36,7 @@ public UaVBehavior(String id, String type, PVector initialPosition, double reaso
 	 * Process the action in the queue to update the speedVal and compassAngle
 	 */
 	protected void processAgentActions(){
-		List<String> toexec = agentState.getAllActions();   
+		List<String> toexec = agentState.getAllActions();
```
Comment (pgavigan) [inline saviSimulation/savi/jason_processing/UaVBehavior.java]: Do we need this change?

Comment (gtrabes) [reply saviSimulation/savi/jason_processing/UaVBehavior.java]: Nope, it's the same line of code

File: saviSimulation/config.cfg (hunk 1 above)
Comment (gtrabes) [reply saviSimulation/config.cfg]: I rather leave it this way till everything works to make sure the errors are not generating noise in the intended behavior.

**Pull request merged**